nodewatcher/plugins/sge.py

```diff
@@ -35,6 +36,37 @@ def getJobs(hostname):
 
     return _jobs
 
+def queueHasPendingJobs():
```

Comment:
_scheduler._`hasPendingJobs()` seems a better read

Reply:
All methods should be lower_snake_case: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

Reply:
I am keeping the pattern in this file. If there needs to be a change, it should be done as a separate task.